src/protocol/http/org/apache/jmeter/protocol/http/control/AuthManager.java

```diff
@@ -123,7 +123,7 @@
          */
         KERBEROS
     }
-
+    public boolean authManagerIsCleared=false;
```

Comment:
This is not needed so it must be removed. I guess it has been added for tests. A better alternative is to fill kerberosManager#subjects and ensure it's clean after that